test/libsolidity/InlineAssembly.cpp

```diff
@@ -747,8 +747,8 @@ BOOST_AUTO_TEST_CASE(shift_constantinople_warning)
 
 BOOST_AUTO_TEST_CASE(jump_error)
```

Comment:
Actually probably most of the tests from this file can be removed since they are available under `syntaxTests/inlineAssembly`.

Reply:
You want to remove them in this PR, or we create some other issue to cleanup tests?

Reply:
I would say clean up test duplication in a separate PR  there is probably a lot kore what can be done. This PR is already getting quite big.